packages/sysutils/lirc/package.mk

```diff
@@ -37,6 +37,13 @@ PKG_CONFIGURE_OPTS_TARGET="ac_cv_func_forkpty=no \
                            --with-gnu-ld \
                            --without-x"
 
+pre_configure_target() {
+  # patch lirc-make-devinput to use target kernel include
+  sed -i \
```

Comment:
mind changing this sed statement so it is more readable? like
```
  sed -e "s|/usr/include/linux/|${SYSROOT_PREFIX}/usr/include/linux/|g" \
      -i ${ROOT}/${PKG_BUILD}/tools/lirc-make-devinput
```
It's just a nitpick :P